plugin/outputStream.go

```diff
@@ -41,13 +41,13 @@ func (s *OutputStream) Write(data []byte) (int, error) {
 	return len(data), err
 }
 
-// Close ensures the channel is closed when the last OutputStream is closed.
-func (s *OutputStream) Close() {
+// close ensures the channel is closed when the last OutputStream is closed.
+func (s *OutputStream) close() {
```

Comment:
Output streams should be closed by calling methods of the `RunningCommand` object, so I don't see a good reason to continue exporting these methods.